app/client/cli/keys.go

```diff
@@ -0,0 +1,465 @@
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				addresses, _, err := kb.GetAll()
+				if err != nil {
+					return err
+				}
+
+				for _, addr := range addresses {
+					fmt.Println(addr)
+				}
+
+				return kb.Stop()
+			},
+		},
+		{
+			Use:     "Get <addrHex>",
+			Short:   "Get <addrHex>",
+			Long:    "Get the keypair <addrHex> from the keybase",
+			Aliases: []string{"get"},
+			Args:    cobra.ExactArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Unpack CLI args
+				addrHex := args[0]
+
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				kp, err := kb.Get(addrHex)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("Address: %s\nPublic Key: %s\n", addrHex, kp.GetPublicKey().String())
+
+				return kb.Stop()
+			},
+		},
+	}
+	return cmds
+}
+
+func keysExportCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Export <addrHex>",
+			Short:   "Export <addrHex>",
+			Long:    "Exports the private key of <addrHex> as a raw or JSON encoded string",
+			Aliases: []string{"export"},
+			Args:    cobra.ExactArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Unpack CLI args
+				addrHex := args[0]
+
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				pwd = readPassphrase(pwd)
+
+				// Determine correct way to export private key
+				var exportString string
+				exportAs = strings.ToLower(exportAs)
+				if exportAs == "json" {
+					exportString, err = kb.ExportPrivJSON(addrHex, pwd)
+					if err != nil {
+						return err
+					}
+				} else if exportAs == "raw" {
+					exportString, err = kb.ExportPrivString(addrHex, pwd)
+					if err != nil {
+						return err
+					}
+				} else {
+					return fmt.Errorf("invalid export format: got %s, want [raw]/[json]", exportAs)
+				}
+
+				if err := kb.Stop(); err != nil {
+					return err
+				}
+
+				// Write to stdout or file
+				if outputFile == "" {
+					fmt.Println(exportString)
+					return nil
+				}
+
+				return writeOutput(exportString, outputFile)
+			},
+		},
+	}
+	return cmds
+}
+
+func keysImportCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Import <privateKeyString>",
+			Short:   "Import <privateKeyString>",
```

Comment:
In `Short`, try to just say "Imports a string or file into the keybase" rather than repeating `Use`.

Ditto elsewhere